File: library/src/main/java/at/htl/neudorfer/entity/Author.java
package at.htl.neudorfer.entity;

import lombok.Data;

@Data
public class Author {

    private String fullName;
//    private String firstName;
//    private String lastName;

    public Author(String fullName) {
        this.fullName = fullName;
        //this.firstName = fullName.split(" ")[0];
        //this.lastName = fullName.split(" ")[1];
    }
}

File: old_books/books/app/src/main/java/at/htl/neudorfer/books/data/models/Author.kt
package at.htl.neudorfer.books.data.models

import androidx.room.Entity
import androidx.room.PrimaryKey
import at.htl.neudorfer.books.utils.Constants

@Entity(tableName = Constants.DATABASE_TABLE_AUHTOR)
data class Author(
    @PrimaryKey(autoGenerate = true)
    val id: Int = 0,
    val firstName: String,
    val lastName: String
)
{}